### DISCUSSION TITLE
Compressed code chunk being deprecated with 1.0?

### DISCUSSION THREAD

**Discussion by blinry** [nesbox/TIC-80#1877] [General]
Heya, quick remark: the [wiki](https://github.com/nesbox/TIC-80/wiki/.tic-File-Format) mentions that the ZIP code chunk in the .tic file format will be deprecated with v1.0.

I just wanted to mention that it's currently really important for sizecoders in the demoscene, like the ones participating in [Lovebyte](https://lovebyte.party), to yield cart sizes that are as small as possible. So I'd argue that, having a real use case, it should not be deprecated. :) What do you think, @joshgoebel?

**Comment by joshgoebel** [comment]
Personally I'm not super sympathetic to this.  If the file format changes then jams _should_ simply update their requirements to match or accept contributions in a overarching ZIP archive or something.  IE, the _requirements should change_. 

But it's up to @nesbox not me, I'm just a contributor.

**Comment by nesbox** [comment]
The compressed code chunk is deprecated, but you can use it if you need to, it won't be removed.
